src/components/NcListItem/NcListItem.vue

```diff
@@ -754,6 +754,11 @@ export default {
 		background-color: var(--color-background-hover);
 	}
 
+	&:focus-visible {
+		outline: 2px solid var(--color-main-text);
+		border: 2px solid var(--color-main-background);
+	}
```

Comment:
Can we make it inside the list item? like `outline-offset: -2`

There are always 2 annoying issues:
1. Outline of one item overlaps to the previous/next ones
2. Outline of the first/last items is cut by `overflow:scroll/hidden` container

Both issues can be solved by adding margins around. However, the second issue is a problem in a virtual list where we need to take into account this additional 4px. It is not possible with external libraries...

Reply:
See: https://github.com/nextcloud/spreed/pull/10818

Reply:
Adjusted it to have a consistent size, what do you think?

https://github.com/nextcloud-libraries/nextcloud-vue/assets/1855448/3b0da021-b818-4ce1-aaf1-898e3b82e319